torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -243,6 +244,19 @@ void GpuLower::replaceSymbolicSizes() {
   }
 }
 
+void GpuLower::collectPaddedParallelDims() {
+  auto used_vals = fusion_->usedMathVals();
+  for (auto tv : ir_utils::filterByType<TensorView>(used_vals)) {
+    for (auto id : tv->domain()->domain()) {
+      if (id->hasPaddingToMultipleOfWarp()) {
+        TORCH_INTERNAL_ASSERT(
+            id->isParallelized(), "Padded serial types not supported");
```

Comment:
Sure. Was expecting extension on the warp patterns in the future but this would be easy enough to update at that point. I just changed it to validate on `TIDx` for now.